bin/docbookrx

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env ruby
 
-if File.exist?(app_lib_path = File.join(File.dirname(__FILE__), '../lib/docbookrx'))
+
+if File.exist?(app_lib_path = File.expand_path(File.join(File.dirname(__FILE__), '../lib/docbookrx')))
```

Comment:
Aha! It appears that it depends how you invoke it. If you add a leading `./`, then it works.

    $ ./bin/asciidoctor

I always run scripts with `./` at the beginning, which is why I never noticed it. However, I suppose it should work either way (if you agree). If so, then we'll update it everywhere.